src/import/papyrus.ts

```diff
@@ -0,0 +1,129 @@
+// Papyrus works with a Queue system, because it is multithreaded, but only one thread can exist per script.
+// (And we can only profile one script at a time.) We parse most "QUEUE_PUSH" events, specifically those that come
+// directly before their corresponding "PUSH" event. Other "QUEUE_PUSH" (and all "QUEUE_POP") events come, as far as I
+// can tell, from Events and are thus asynchronous. They are ignored.
+
+// Stack profiling also puts a START operation at the top and a STOP operation at the bottom of the file. These are
+// completely useless to us.
+
+// Stack profiling also logs the Form a method is run on. For those that are not familiar with
+// Papyrus terminology, a "Form" is an instance of a type defined by a script. E.g. a quest is a form that extends
+// the "Quest" script, and thus it has certain methods, like "CompleteQuest()". This information would be useful
+// for Debugging, but for profiling, it would hinder meaningful output in left heavy mode.
```

Comment:
Thank you for including a top-of-file comment like this! One small nice addition would be to explain what Papyrus is, since it's relatively obscure